frappe/email/doctype/email_domain/email_domain.py

```diff
@@ -55,7 +56,7 @@ def validate(self):
 						test = poplib.POP3(self.email_server, port=get_port(self))
 
 			except Exception as e:
-				logger.warn(
+				logger.warning(
```

Comment:
can this be changed in other places in this file as well? Thanks

Reply:
Yes another place fixed